retriever/lib/engine.py

```diff
@@ -4,12 +4,7 @@
 from future import standard_library
 
 standard_library.install_aliases()
-from builtins import object
-from builtins import range
-from builtins import input
-from builtins import zip
-from builtins import next
-from builtins import str
+from builtins import object, range, input, zip, next, str
```

Comment:
Note: The changes in this file, from here till line [455](https://github.com/weecology/retriever/pull/1142/files#diff-ff05cd5f49d751e66a61636b498c70acR455), are related to either refactoring and python idioms.